README.md

```diff
@@ -39,6 +39,7 @@ const url = await run("itemId", options?);
 | dpi        | number  | The DPI to use when rendering a map print. Defaults to `96`.                                                                                                                      |
 | parameters | object  | An object specifying additional parameters to pass to the job.                                                                                                                    |
 | portalUrl  | string  | The URL of the ArcGIS Portal instance to use. Defaults to ArcGIS Online: `"https://www.arcgis.com"`.                                                                              |
+| resultName | string  | The desired name of the output file. |
```

Comment:
Updated to match the JSDoc text.

Reply:
@ianschmitz Do you think we should provide a default? The GXP/GXR service will happily continue with its current behavior if no name is specified.

Reply:
I don't think we necessarily need to provide a default value in this library. For example we specified `dpi defaults to 96` in the docs, even though we don't specify a dpi in the request, as this is the default on the server. That could be seen as a bad idea though because if we ever changed the default on the server these docs would be out of date. Thoughts?

Reply:
There is no default name provided by the printing/reporting service. When a name is not provided in our SaaS environment, the file name will continue to match the name of the blob in Azure. This is an ugly name that id GUID-like and was part of the motivation for adding the ability to specify a name. With an on prem installation and no name provided, the browser tab name and suggested download name are determined by the browser (eg. Chrome uses 'Result').
I'm inclined to leave the documentation as is and not mention anything about a default name.